### PR TITLE
Maproulette

### PR DESCRIPTION
**Pull request by gknisely**

### CODE REVIEW

**Comment by Kopkins** [comment]
Look like it has everything `admin_tool.py` needs to run, just need to make sure that python3 is installed on the box

**Comment by gknisely** [comment]
Should be all set....can someone look at this please.

File: attributes/default.rb [hunk 1]
```diff
@@ -35,6 +35,17 @@
...
+#map roulette
+default[:maproulette][:config_file]                              = 'maproulette.json'
+default[:maproulette][:geojson_file]                             = 'maproulette_tasks.geojson'
+default[:maproulette][:recurring_tasks_file]                     = 'recurring_tasks_file.txt'
+default[:maproulette][:server_url]                               = 'http://localhost:9000'
+default[:maproulette][:run_admin_tool]                           = false
+default[:maproulette][:admin_dir]                                = "#{node[:valhalla][:src_dir]}/mjonir/py/challenge_admin"
+default[:maproulette][:api_key]                                  = 'YOUR_KEY'
+default[:maproulette][:config]                                   = "#{node[:valhalla][:conf_dir]}/#{node[:maproulette][:config_file]}"
+default[:maproulette][:geojson]                                  = "#{node[:valhalla][:tile_dir]}/#{node[:maproulette][:config_file]}"
```
Comment (kevinkreiser) [inline attributes/default.rb]: copy paste fail

File: templates/default/cut_tiles.sh.erb [hunk 2]
```diff
@@ -68,6 +68,11 @@ mv_stamp maproulette_tasks.geojson ${stamp}
 cp_stamp <%= node[:valhalla][:transit_dir] %>.tgz ${stamp}
 popd
 
+# do we want to run map roulette admin tool
+if [ "<%= node[:maproulette][:run_admin_tool] %>" == true ]; then
+    <%= node[:maproulette][:admin_dir] %>/admin_tool.py -c <%= node[:maproulette][:config] %> -i <%= node[:maproulette][:geojson] %>
```
Comment (kevinkreiser) [inline templates/default/cut_tiles.sh.erb]: 2 too many spaces

**Comment by gknisely** [comment]
@kevinkreiser one more :heavy_check_mark: please

**Comment by Kopkins** [comment]
:ship: :shipit: :rocket:

**Pull request merged**